client/src/cmdhficlass.c

```diff
@@ -1528,10 +1499,10 @@ static int CmdHFiClassDump(const char *Cmd) {
         .req.use_credit_key = false,
         .req.use_replay = use_replay,
         .req.send_reply = true,
-        .req.do_auth = auth,
+        .req.do_auth = have_debit_key,
```

Comment:
I think you missed the user-case  credit key.   The old auth was true for either debit or credit

Reply:
ah, okay I can rename it back.  I was just trying to make things follow similar naming conventions as auth was generic

Reply:
okay, I see what I did there.  Should be fixed in the latest commit.